src/Config.cxx

```diff
@@ -29,9 +20,6 @@ Listener::Listener(KafkaW::BrokerOpt bopt, uri::URI uri) {
         }
         impl->cv.notify_all();
       };
-  consumer.on_rebalance_assign = {};
-  consumer.on_rebalance_start = {};
-  consumer.add_topic(uri.topic);
 }
 
 Listener::~Listener() {}
```

Comment:
This line can be removed, put `~Listener() = default;` in the header file instead.